doc/tutorials/acrn_configuration_tool.rst

```diff
@@ -373,6 +399,13 @@ current scenario has:
   ``bus#-port#[:bus#-port#: ...]``, e.g.: ``1-2:2-4``.
   Refer to :ref:`usb_virtualization` for details.
 
+``shm_regions``:
+  List of shared memrory regions for inter-VM communication.
+
+``shm_region`` (a child node of ``shm_regions``):
+  configure the shm regions for current VM, input format: hv:/<;shm name>;,
```

Comment:
Why is `dm:/` not mentionned in this section?

Reply:
We don't need to list the VM IDs here?

Reply:
> Why is `dm:/` not mentionned in this section?
the dm land solution for IVSHMEN is not officially supported in config tool, so remove dm:/ in description of IVSHMEM region and here.

> We don't need to list the VM IDs here?
Yes. It is by design of IVSHMEM.

Reply:
Are you saying we only support IVSHMEM hv-land solutions via XML configuration, and only support IVSHMEM dm-land solutions via acrn-dm command line parameters?  We need to clarify this in the documentation.  @NanlinXie

Reply:
> Are you saying we only support IVSHMEM hv-land solutions via XML configuration, and only support IVSHMEM dm-land solutions via acrn-dm command line parameters? We need to clarify this in the documentation. @NanlinXie

we only support hv-land solutions, I have remove dm-land description in my update.